parser/parser.coffee

```diff
@@ -126,6 +127,18 @@ parseClassExpression = ->
   new Class expressions
 
 
+parseTraitExpression = ->
+  match 'trait'
+  match ':'
+  match 'newline'
+  expressions = []
+  while not at 'end'
+    expressions.push parseExpression()
```

Comment:
We really need to have a discussion about this since it allows for nested classes/traits.